feature/eventdetail/build.gradle.kts

```diff
@@ -0,0 +1,86 @@
+plugins {
+    kotlin("multiplatform")
+    id("com.android.library")
+    id("org.jetbrains.kotlin.plugin.parcelize")
+    alias(libs.plugins.kotlin.compose)
+    alias(libs.plugins.cash.paparazzi)
+}
+
+kotlin {
+    android()
+    jvm()
+
+    sourceSets {
+        val commonMain by getting {
+            dependencies {
+                implementation(project(":core:displaymodels"))
+                implementation(project(":core:feature"))
+                implementation(project(":core:models")) // I still don't think this is necessary
+                implementation(project(":data:event"))
+                implementation(project(":data:match"))
+                implementation(project(":shared:design-system"))
+                implementation(project(":shared:ui"))
+                implementation(compose.foundation)
+                implementation(compose.material3)
+                implementation(compose.materialIconsExtended)
+                implementation(compose.runtime)
+                implementation(libs.koin.core)
+                implementation(libs.slack.circuit)
```

Comment:
Look into:

1. Bundling the compose deps
2. Creating a feature bundle for all of these deps.